test/e2e/functional/default-pdb-support.yaml

```diff
@@ -0,0 +1 @@
+../../../examples/default-pdb-support.yaml
```

Comment:
this suggests you have a link here - I'd suggest you copy the file instead so you can add the argo-e2e label